src/extension/app/constants.js

```diff
@@ -137,4 +155,35 @@ export const ICONS = {
         <path d="M6.71775 17.1971C7.17047 17.1971 7.4639 16.8991 7.4639 16.4687C7.4639 16.2535 7.38845 16.0962 7.26269 15.9637L4.83978 13.6379L2.76898 11.8831L2.61808 12.2638L5.21705 12.3797H15.9231C17.8766 12.3797 18.6898 11.4941 18.6898 9.62336V4.7315C18.6898 2.8029 17.8766 2 15.9231 2H11.2283C10.7672 2 10.4653 2.33109 10.4653 2.73668C10.4653 3.14227 10.7672 3.47335 11.2283 3.47335H15.9231C16.8119 3.47335 17.1891 3.85411 17.1891 4.7315V9.62336C17.1891 10.5256 16.8119 10.9063 15.9231 10.9063H5.21705L2.61808 11.014L2.76898 11.3947L4.83978 9.6482L7.26269 7.32229C7.38845 7.18985 7.4639 7.0243 7.4639 6.80909C7.4639 6.38695 7.17047 6.08069 6.71775 6.08069C6.54169 6.08069 6.3237 6.17175 6.18118 6.31246L1.25151 11.0885C1.08384 11.2457 1 11.4361 1 11.6431C1 11.8416 1.08384 12.0403 1.25151 12.1976L6.18118 16.9736C6.3237 17.1143 6.54169 17.1971 6.71775 17.1971Z" fill="currentColor"/>
       </g>
     </svg>`,
+  INFO: html`
+    <svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg" >
+      <path d="M10 18.75C5.1748 18.75 1.25 14.8252 1.25 10C1.25 5.1748 5.1748 1.25 10 1.25C14.8252 1.25 18.75 5.1748 18.75 10C18.75 14.8252 14.8252 18.75 10 18.75ZM10 2.75C6.00195 2.75 2.75 6.00195 2.75 10C2.75 13.998 6.00195 17.25 10 17.25C13.998 17.25 17.25 13.998 17.25 10C17.25 6.00195 13.998 2.75 10 2.75Z" fill="currentColor"/>
+      <path d="M10.0006 5.26033C10.2313 5.2522 10.456 5.3342 10.6273 5.48895C10.9576 5.854 10.9576 6.40997 10.6273 6.77502C10.4579 6.93353 10.2324 7.0181 10.0006 7.01006C9.76426 7.01954 9.53472 6.92971 9.36759 6.76231C9.20552 6.59441 9.11843 6.36799 9.12622 6.13476C9.11384 5.89979 9.19581 5.66961 9.35392 5.49536C9.5275 5.33062 9.76179 5.24548 10.0006 5.26033Z" fill="currentColor"/>
+      <path d="M10 15.0625C9.58594 15.0625 9.25 14.7266 9.25 14.3125V9.47754C9.25 9.06348 9.58594 8.72754 10 8.72754C10.4141 8.72754 10.75 9.06348 10.75 9.47754V14.3125C10.75 14.7266 10.4141 15.0625 10 15.0625Z" fill="currentColor"/>
+    </svg>`,
+  CHECKMARK: html`
+    <svg width = "20" height = "20" viewBox = "0 0 20 20" fill = "none" xmlns = "http://www.w3.org/2000/svg" >
+      <path d="M10 18.75C5.1748 18.75 1.25 14.8252 1.25 10C1.25 5.1748 5.1748 1.25 10 1.25C14.8252 1.25 18.75 5.1748 18.75 10C18.75 14.8252 14.8252 18.75 10 18.75ZM10 2.75C6.00195 2.75 2.75 6.00195 2.75 10C2.75 13.998 6.00195 17.25 10 17.25C13.998 17.25 17.25 13.998 17.25 10C17.25 6.00195 13.998 2.75 10 2.75Z" fill="white"/>
+      <path d="M9.18281 13.9434C8.9709 13.9434 8.76777 13.8535 8.62519 13.6953L5.98164 10.7559C5.7043 10.4473 5.72969 9.97363 6.0373 9.69629C6.34491 9.41895 6.81855 9.44434 7.09687 9.75195L9.12324 12.0059L12.8234 6.95996C13.0666 6.62695 13.5354 6.55078 13.8713 6.79883C14.2053 7.04297 14.2775 7.5127 14.0324 7.84668L9.7873 13.6367C9.65449 13.8193 9.4455 13.9316 9.21992 13.9424C9.2082 13.9434 9.19551 13.9434 9.18281 13.9434Z" fill="white"/>
```

Comment:
Should fill color be `currentColor`?

Reply:
Currently it's always going to be white but who knows where we are going to use it. Will change